lib/plugins/aws/utils/resolveCfImportValue.js

```diff
@@ -7,7 +7,10 @@ function resolveCfImportValue(provider, name, sdkParams = {}) {
     if (result.NextToken) {
       return resolveCfImportValue(provider, name, { NextToken: result.NextToken });
     }
-    return null;
+
+    throw new Error(
```

Comment:
I think we need to throw _operational_ error here (so `ServerlessError`), we can require it from `classes/Error`